inventory/technical/inventory_valuation.rst

```diff
@@ -0,0 +1,95 @@
+===============================
+Inventory Valuation In Detail
+===============================
+
+The following chapter is more technical and explains how specific elements are computed.
+Basically the elements below are computed for a product.product not for a product.template.
+When calculated for different elements, it may differs.
+
+On Hand quantities
+*******************
+
+- For product.template
+
+Sum of the quantities of all the product.product (variants)
+
+- For product.product
+
+TODO:
+
+
+Inventory Valuation (Current Inventory)
+*****************************************
+This is specified in odoo/addons/stock_account/models/product.py mostly in the method _compute_stock_value() (see refA_)
```

Comment:
can you add a link to the source file on GitHub directly, it would make the documentation easier to browse

Reply:
not sure how to do it.
Further, I'm afraid about code changes, If I target L100 of a file but someone adds 10 lines at the beginning, would my link be still up to date?

Whatever I just update and make something readable.